src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigCol {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(double base1, double base2, double height, Color color) {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the beginning of the constructor.